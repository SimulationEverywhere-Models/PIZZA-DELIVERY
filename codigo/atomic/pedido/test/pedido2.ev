00:00:01:000 Pedido_in 1
00:00:02:000 Esperar_p 1
00:00:03:000 Pedido_in 2
00:00:04:000 Pedido_in 3
00:00:05:000 Pedido_in 4
00:00:06:000 Pedido_in 5
00:00:07:000 Pedido_in 6
00:00:08:000 Listo_p 1
