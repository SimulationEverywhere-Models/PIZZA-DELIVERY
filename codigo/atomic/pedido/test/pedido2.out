00:00:01:030 pedido_out 1
00:00:07:000 parar_p 1
00:00:08:000 pedido_out 2
00:00:08:000 libre_p 1
00:00:08:030 pedido_out 3
00:00:08:060 pedido_out 4
00:00:08:090 pedido_out 5
00:00:08:120 pedido_out 6
